TEP001_extensive_test_suite.rst

```diff
@@ -29,36 +29,27 @@ TRAVIS and for swift development.
 Ensuring to enable running of full-tests in a useful way this TEP aims to
 work on two major issues:
```

Comment:
Well, let's double check the grammar and the main point of this "sentence" please